clients/js/src/plugin.ts

```diff
@@ -35,6 +35,7 @@ import {
   createCivicGatewayProgram,
   createMplTokenAuthRulesProgram,
 } from './programs';
+import { allocationGuardManifest } from './defaultGuards/allocation';
```

Comment:
Can you import that as part of the group import on line 24 instead?

Reply:
Done.